drivers/ili9341/include/ili9341_params.h

```diff
@@ -95,7 +98,7 @@ extern "C" {
 /**
  * @brief   Configure ILI9341
  */
-static const lcd_params_t ili9341_params[] =
+static lcd_params_t ili9341_params[] =
```

Comment:
Actual SPI frequency should be stored in `lcd_t` instead of making the parameter set non-const.